index.html

```diff
@@ -1,8 +1,118 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
   </head>
-
   <body>
+    <section class="background">
+      <header>
+        <ul>
+          <li>central pennsylvania</li>
+          <li>open source conference</li>
+        </ul>
+      </header>
+        <section class="hero">
+          <h1>Into the Great Wide Open (Source)</h1>
+        </section>
+    </section>
+    <section>
```

Comment:
Make sure you class everything before it's added to github.

Reply:
i figured we can add more classes once we start css

Reply:
Thats fine I just didn't know that was your style